### DISCUSSION TITLE
Start pimcore Controller from the commandline

### DISCUSSION THREAD

**Discussion by incoxx** [pimcore/pimcore#10193] [General]
**General question:**

For a data object preview, if have a Pimcore controller and a template/view

Using static routes i can call the controller/view using an url and see the result in the browser window.

Is it possible to call a controller/view with parameters from the Linux shell (like a symphony command)?

Sample command from the shell (if it´s possible): 
Execute controller mycontroller with view myview and parameter id=0815

php mycontroller myview id=0815 >> out.html 

I know i can call the pimcore url with wget or curl.  

Or is there a simple way to call a controller/view using a pimcore command as a "wrapper"?

**Comment by dpfaffenbauer** [comment]
why would you wanna do that? just use curl/wget?!
